tests/parser/syntax/test_utils.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from pytest import raises
+
+from vyper import compiler
+from vyper.exceptions import VariableDeclarationException
+
+fail_list = [
+    """
+@public
+def foo(i: int128) -> int128:
+    ő1qwerty : int128 = i
+    return ő1qwerty
+    """,
+    """
+@public
+def foo(i: int128) -> int128:
+    fő1234 : int128 = i
```

Comment:
the first four tests all check the same condition anyway, you can get rid of three of them!